docs/src/recovery.md

```diff
@@ -87,6 +87,11 @@ spec:
           maxParallel: 8
 ```
 
+The previous example assumes that the application database and its owning user
+are named `app` by default. If the PostgreSQL cluster being restored uses
+different names upon exiting the recovery phase, you must specify these names
```

Comment:
IMO the comma is needed here. Also, "upon" is not preventative
```suggestion
different names, upon exiting the recovery phase, you must specify these names
```

Might even rephrase:
```suggestion
different names, you must specify these names before exiting the recovery phase,
```